aws-xray-agent-plugin/src/test/java/com/amazonaws/xray/agent/runtime/handlers/downstream/AWSHandlerIntegTest.java

```diff
@@ -118,17 +121,12 @@ public void testDynamoDBListTable() {
     @Test
     public void testSQSSendMessage() {
         AmazonSQS sqs = (AmazonSQS) createTestableClient(AmazonSQSClientBuilder.standard()).build();
-        // XML acquired by intercepting a valid AWS SQS response.
-        String result = "<SendMessageResponse xmlns=\"http://queue.amazonaws.com/doc/2012-11-05/\">\n" +
-                "\t<SendMessageResult>\n" +
-                "\t\t<MessageId>de9e2f1b-aa00-43a6-84b8-b5379085c0f2</MessageId>\n" +
-                "\t\t<MD5OfMessageBody>c1ddd94da830e09533d058f67d4ef56a</MD5OfMessageBody>\n" +
-                "\t</SendMessageResult>\n" +
-                "\t<ResponseMetadata>\n" +
-                "\t\t<RequestId>41edd773-d43f-5493-b43b-814e965a23f1</RequestId>\n" +
-                "\t</ResponseMetadata>\n" +
-                "</SendMessageResponse>";
-        mockHttpClient(sqs, result);
+        String result = "{" +
+            "\"MD5OfMessageAttributes\": \"c48838208d2b4e14e3ca0093a8443f09\"," +
+            "\"MD5OfMessageBody\": \"c1ddd94da830e09533d058f67d4ef56a\"," +
+            "\"MessageId\": \"de9e2f1b-aa00-43a6-84b8-b5379085c0f2\"" +
+        "}";
```

Comment:
Tests break after for aws-java-sdk `v1.12.584+` (works on `<=v1.12.583`).
See:
- https://github.com/aws/aws-sdk-java/discussions/3114